src/hooks/api/faq/useCreateFaq.ts

```diff
@@ -0,0 +1,13 @@
+import { useMutation } from '@tanstack/react-query';
+import { createFaq } from '@/apis/index';
+
+export const useCreateFaq = () => {
+  return useMutation(createFaq, {
+    onSuccess: () => {
+      console.log('FAQ 생성성공');
```

Comment:
console은 개발 편의를 위해서 생성하신거라고 생각해요. console을 제거하고 toast를 여기에 띄우는 것은 어떨까요?
```suggestion
       toast.success('FAQ가 성공적으로 저장되었습니다');
```

Reply:
토스트로 변경하였습니다 ! 감사합니다 !